phase1/vsphere/README.md

```diff
@@ -48,16 +49,141 @@ cd kubernetes-anywhere
 ### Enter the `kubernetes-anywhere` deployment environment:
 
 ```shell
-make docker-dev
+make docker-dev # The first run of this will build the docker image to be used and can take a few minutes.
 ```
 
 ### Start the deployment wizard:
 
 ```shell
 make deploy
 ```
+
 and fill complete the config wizard to deploy a kubernetes-anywhere cluster.
 
+Sample config below
+
+* Select the number of nodes. Master + Number of nodes will be deployed.
+```
+*
+* Phase 1: Cluster Resource Provisioning
+*
+number of nodes (phase1.num_nodes) [4] (NEW) 8
+```
+
+* Set the cluster name. A folder with the cluster name will be created to place all the VMs.
+```
+cluster name (phase1.cluster_name) [kubernetes] (NEW) k8s
+```
+
+* Select the provider
+```
+*
+* cloud provider: gce, azure or vsphere
+*
+cloud provider: gce, azure or vsphere (phase1.cloud_provider) [gce] (NEW) vsphere
+```
+
+* Set the vCenter URL
+```
+  *
+  * vSphere configuration
+  *
+  vCenter URL Ex: 10.192.10.30 or myvcenter.io (phase1.vSphere.url) [] (NEW) 10.192.72.70
+```
+
+* Set the port for vCenter communication. Unless vCenter is setup with a different port select the default port.
+```
+  vCenter port (phase1.vSphere.port) [443] (NEW) 
+```
+
+* Set the user name for vCenter. All vCenter operations will be performed using these user credentials
+```
+  vCenter username (phase1.vSphere.username) [] (NEW) administrator@myvcenter.io
+```
+
+* Set the password for vCenter.
+```
+  vCenter password (phase1.vSphere.password) [] (NEW) MyPassword#3
+```
+
+* Set the type of certificate used by vCenter. Set to true for self signed certificates
+```
+  Does host use self-signed cert (phase1.vSphere.insecure) [true] (NEW) 
+```
+
+* Set the datacenter in vCenter to use
+```
+  Datacenter (phase1.vSphere.datacenter) [datacenter] (NEW) PA-DC
+```
+
+* Set the datastore to be use. This will be used for placing the VMs and volumes created via storage classes/dynamic provisioning.
+```
+  Datastore (phase1.vSphere.datastore) [datastore] (NEW) vsanDatastore
+```
+
+* Set the resource pool. This should be the same as the one to which the template OVA was imported. To get the default resource pool run
+```govc ls -l 'host/*' | grep ResourcePool | awk '{print $1}' ```. [Resource Pool Documentation](https://pubs.vmware.com/vsphere-65/index.jsp?topic=%2Fcom.vmware.vsphere.resmgmt.doc%2FGUID-60077B40-66FF-4625-934A-641703ED7601.html) and [DRS Cluster Documentation](https://pubs.vmware.com/vsphere-65/index.jsp#com.vmware.vsphere.resmgmt.doc/GUID-8ACF3502-5314-469F-8CC9-4A9BD5925BC2.html)
+
+```
+  Resource pool/cluster (same as destination for OVA import). (phase1.vSphere.resourcepool) [] (NEW) /PA-DC/host/10.20.104.41/Resources
```

Comment:
The default Resource pool needs the hierarchy.. unless a cluster is created... it is confusing.. will dig through more to see if this can be made simpler but there are 2 answers here a hierarchy or the cluster name from what I understand.